src/payment/scripts/db-schema.sql

```diff
@@ -1 +1,55 @@
-CREATE TABLE users
+CREATE TABLE tUser (
+	UserID serial PRIMARY KEY,
+	LastName VARCHAR(64) NOT NULL,
+	FirstName VARCHAR(64) NOT NULL,
+	Address VARCHAR(255),
+	City VARCHAR(255),
+	Email VARCHAR(255) UNIQUE NOT NULL,
+	CreditCard VARCHAR(40),
+	created_on TIMESTAMP NOT NULL
+);
+
+CREATE TABLE tReservation (
+	UserID int NOT NULL,
+	ReservationUUID uuid NOT NULL,
+	created_on TIMESTAMP NOT NULL,
+	CONSTRAINT user_id_fkey FOREIGN KEY(UserID) 
+		REFERENCES tUser (UserID) MATCH SIMPLE
+		ON UPDATE NO ACTION ON DELETE NO ACTION
+);
+
+
+CREATE TABLE tCar (
+	CarID serial PRIMARY KEY,
+	ReservationUUID uuid NOT NULL,
+	Agency VARCHAR(255),
+	locationAddress VARCHAR(255),
+	pickupDateTime TIMESTAMP NOT NULL,
+	dropoffDateTime TIMESTAMP NOT NULL,
+	costUSD decimal NOT NULL,
+	created_on TIMESTAMP NOT NULL
+
+);
+
+CREATE TABLE tHotel (
+	HotelID serial PRIMARY KEY,
+	ReservationUUID uuid NOT NULL,
+	HotelName VARCHAR(255),
+	locationAddress VARCHAR(255),
```

Comment:
Currently we have city and country for location data. I am wondering if this should be separated or just put both in here separated by a comma

Reply:
yes this was draft one, location will be city and country 
should we include street address ( like 43 Main Stree ) ?

Reply:
@Grant-Steinfeld We don't currently have addresses so I would say hold off on that

Reply:
OK - merged as I had some new stuff to PR - oddly could not do that from my Fork?